agent-control/tests/k8s/flux_self_update.rs

```diff
@@ -36,6 +36,8 @@ use std::sync::Arc;
 use std::time::Duration;
 use tempfile::tempdir;
 
+const TEST_RELEASE_NAME: &str = "agent-control-cd";
```

Comment:
nit, use something different than the default just in case we catch some issue when is modified